tools/init.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+psql <<EOF
+CREATE EXTENSION plpython3u;
+CREATE EXTENSION rikai;
+
+INSERT INTO ml.models (name, flavor, model_type)
+VALUES ('ssd', 'pytorch', 'ssd');
+
+-- Test data
+-- See images in rikai/tests/conftest.py
+CREATE EXTENSION IF NOT EXISTS pgtap;  -- For unit test
+
+CREATE TABLE images (
+image_id SERIAL PRIMARY KEY,
+image image
+);
+INSERT INTO images (image)
+VALUES
+(Row('http://farm2.staticflickr.com/1129/4726871278_4dd241a03a_z.jpg')),
```

Comment:
want to just pull these 2 images into the repo in case these get moved/deleted/updated?

Reply:
good point
Filed #6